test/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/returns/yeartodatelliability/YearToDateLiabilityControllerSpec.scala

```diff
@@ -2730,8 +2930,12 @@ class YearToDateLiabilityControllerSpec
     oldDraftReturn: DraftReturn,
     updatedDraftReturn: DraftReturn
   ): Unit = {
-    val journey = sample[FillingOutReturn].copy(draftReturn = oldDraftReturn)
-    val session = SessionData.empty.copy(journeyStatus      = Some(journey))
+    val journey = sample[FillingOutReturn].copy(
+      draftReturn       = oldDraftReturn,
+      subscribedDetails = sample[SubscribedDetails].copy(name = sample[IndividualName].asRight[TrustName])
```

Comment:
you save a few characters with just `Right(sample[IndividualName[)` instead of `sample[IndividualName].asRight[TrustName]`

Reply:
will Do. Thanks

Reply:
Done.